docs/installing_server.md

```diff
@@ -626,3 +626,89 @@ that were described previously. The following steps are needed:
         ISS=https://auth.cern.ch/auth/realms/cern" \
     --email "rucio@cern.ch"
     ```
+
+## S3 Storage Configuration (FTS3 & GFAL2)
```

Comment:
I think this should be it's own article in the documentation. (Under Operator documentation)

Reply:
Indeed, fits better there, moving it!